model/src/w3triamd.F90

```diff
@@ -2805,130 +2802,6 @@ SUBROUTINE TRIANG_INDEXES(I, INEXT, IPREV)
-    !/
-    !/    01-June-2018 : Origination.                        ( version 6.04 )
-    !/
-    !  1. Purpose : get interface nodes of the wetting and drying part
-    !  2. Method :
-    !  3. Parameters :
-    !
-    !     Parameter list
-    !     ----------------------------------------------------------------
-    !     ----------------------------------------------------------------
-    !
-    !  4. Subroutines used :
-    !
-    !      Name      Type  Module   Description
-    !     ----------------------------------------------------------------
-    !      STRACE    Subr. W3SERVMD Subroutine tracing.
-    !     ----------------------------------------------------------------
-    !
-    !  5. Called by :
-    !
-    !      Name      Type  Module   Description
-    !     ----------------------------------------------------------------
-    !     ----------------------------------------------------------------
-    !
-    !  6. Error messages :
-    !  7. Remarks
-    !  8. Structure :
-    !  9. Switches :
-    !
-    !     !/S  Enable subroutine tracing.
-    !
-    ! 10. Source code :
-    !
-    !/ ------------------------------------------------------------------- /
-#ifdef W3_S
-    USE W3SERVMD, ONLY: STRACE
-#endif
-    !
-    USE CONSTANTS, ONLY : LPDLIB
-    USE W3GDATMD, ONLY : NX, IOBP, CCON, NSEAL, IOBDP, IE_CELL, IOBDP, TRIGP
-#ifdef W3_PDLIB
-    USE yowNodepool, only: PDLIB_SI, PDLIB_IEN, PDLIB_CCON, NPA, PDLIB_IE_CELL2, PDLIB_POS_CELL2
-    USE yowElementpool, only: INE
-#endif
-    IMPLICIT NONE
-    !/
-    !/ ------------------------------------------------------------------- /
-    !/ Parameter list
-    !/
-    !/ ------------------------------------------------------------------- /
-    !/ Local PARAMETERs
-    !/
-#ifdef W3_S
-    INTEGER, SAVE           :: IENT = 0
-#endif
-    !/
-    !/ ------------------------------------------------------------------- /
-    !/
-    INTEGER :: I, J, IP, IE
-#ifdef W3_S
-    CALL STRACE (IENT, 'GET_INTERFACE')
-#endif
-#ifdef W3_PDLIB
-    IF (LPDLIB) THEN
-      DO IP = 1, NSEAL
-        IF (IOBP(IP) .NE. 0 .OR. IOBDP(IP) .EQ. 0) CYCLE
-        DO I = 1, PDLIB_CCON(IP)
-          IE = PDLIB_IE_CELL2(I,IP)
-          IF (ANY(IOBDP(TRIGP(:,IE)) .EQ. 0)) THEN
-            IOBDP(IP) = -1
-            CYCLE
-          ENDIF
-        ENDDO
-      ENDDO
-      !CALL EXCHANGE_....
-    ELSE
-#endif
-      J = 0
-      DO IP = 1, NSEAL
-        DO I = 1, CCON(IP)
-          J = J + 1
-          IE = IE_CELL(J)
-          IF (ANY(IOBDP(TRIGP(:,IE)) .EQ. 0)) THEN
-            IOBDP(IP) = -1 ! Set this node as a wet node adjacent to a dry one ... now what's next? Here on this points we want to compute the reflection source term, yes?
-            EXIT
-          ENDIF
-        ENDDO
-      ENDDO
-#ifdef W3_PDLIB
-    ENDIF
-#endif
-
-  END SUBROUTINE GET_INTERFACE
-  !/ ------------------------------------------------------------------- /
-
-  !>
```

Comment:
Please do not remove doxygen for SET_UG_IOBP.  Feel free to edit it but we should keep this.

Reply:
done

Reply:
Did the update here get pushed, I still don't see this text as being deleted?

Reply:
The doxygen text for SET_UG_IOBP needs to be restored

Reply:
This has now been fixed.  Thank you.